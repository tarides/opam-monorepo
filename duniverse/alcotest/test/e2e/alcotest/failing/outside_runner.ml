(** Capture the behaviour of the assertion functions when not running inside
    [Alcotest.run]. *)

let () =
  Alcotest.(check int) "" 1 1 (* Empty passing assertion; should be silent *);
  Alcotest.(check int) "Passing assertion" 1 1;
  Alcotest.(check int) "Failing assertion" 1 2
