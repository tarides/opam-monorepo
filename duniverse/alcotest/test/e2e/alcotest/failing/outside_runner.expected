ASSERT Passing assertion
ASSERT Failing assertion
Fatal error: exception Alcotest assertion failure
FAIL Failing assertion

   Expected: `1'
   Received: `2'


